spec/send_sms_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'send_sms'
+
+describe Message do
+  it 'should instantiate a new instance' do
+    expect(Message.new).to be_instance_of Message
```

Comment:
As shared in the slack channel this morning, https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#stubbing-the-twilio-api is a resource you may find helpful 🙂